src/tag-input/src/TagInput.js

```diff
@@ -51,10 +53,10 @@ class TagInput extends React.Component {
      */
     onRemove: PropTypes.func,
     /** Value or RegExp to split on pasted text or on enter keypress */
-    separator: PropTypes.oneOf([
+    separator: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.instanceOf(RegExp),
-      false
+      PropTypes.oneOf([false])
```

Comment:
Couldn't find a better way to do this. `false` is allowed but `true` isn't.

Reply:
lgtm